External/Plugins/CodeRefactor/Commands/OrganizeImports.cs

```diff
@@ -226,7 +225,24 @@ class ImportsComparerType : IComparer<MemberModel>
     {
         public Int32 Compare(MemberModel item1, MemberModel item2)
```

Comment:
Having never used C# before, for some reason I couldn't make the class from one plug-in visible in the other. But on second try it worked, maybe I've made a typo or something. I've moved the comparison code to a new ByImportTypeMemberComparer class in MemberModel.cs. OrganizeImports uses this member comparer for sorting, however ASGenerator.cs does regex-based matching so it needs to compare strings, not members, so I exposed the string comparison function as a static function ByImportTypeMemberComparer.CompareImports
I'm not sure it's the best place for this class, feel free to move it somewhere else.